typedoc.json

```diff
@@ -1,6 +1,7 @@
 {
   "out": "./docs",
   "readme": "README.md",
+  "excludePrivate": true,
   "exclude": [
     "src/signalingprotocol/SignalingProtocol.js",
     "src/signalingprotocol/SignalingProtocol.d.ts",
```

Comment:
Since these two files are excluded should we make changes to them?

Reply:
They are still processed by analysis tools like api-extractor, so I fixed them.

Reply:
Got it, so no configuration on the api-extractor level in that case?

Reply:
We'll get there when we get there :joy: